app/Pages/ContactPage.tsx

```diff
@@ -1,9 +1,61 @@
-import { View } from "react-native";
-import { Text, useTheme } from "react-native-paper";
+import React from "react";
+import { View, useWindowDimensions } from "react-native";
+import {
+  Button,
+  Card,
+  SegmentedButtons,
+  Text,
+  TextInput,
+  useTheme,
+} from "react-native-paper";
 import { SafeAreaProvider, SafeAreaView } from "react-native-safe-area-context";
+import { CARD_IMAGE_HEIGHT, CARD_PADDING } from "../Components/Home/Cards";
+import Alert from "react-native-awesome-alerts";
+import { rgba } from "polished";
+import { useNavigation } from "@react-navigation/native";
 
 const ContactPage = () => {
   const theme = useTheme();
+  const navigation: any = useNavigation();
+  const size = useWindowDimensions();
+  const [value, setValue] = React.useState("");
+  const [text, setText] = React.useState("");
+  const [label, setLabel] = React.useState("0/1000");
+  const [disabled, setDisabled] = React.useState(true);
+  const [placeholder, setPlaceholder] = React.useState("");
+  const [thankYouVisible, setThankYouVisible] = React.useState(false);
+  const [buttonText, setButtonText] = React.useState("Submit Feedback*");
+  const [errorVisible, setErrorVisible] = React.useState(false);
```

Comment:
I used to do this a lot, having individual useStates for each value. I think creating a state object makes things easier to understand / maintain. 
```typescript
interface contactPageState {
  value: string
  text: string
  label: string
  placeholder: string
  buttonText: string
  buttonDisabled: boolean
}
const [contactPage, setContactPage] = React.useState<contactPageState>({
  buttonDisabled: true,
  buttonText: "Submit Feedback*",
  label: "0/1000",
  text: "",
  placeholder: "",
  value: ""
});
```

For retrieving a state would use ```contactPage.value```
And for updating a state we can do:
```typescript
setContactPage({
  ...contactPage,
  value: value,
  disabled: false,
  placeholder: placeholder,
});
```